controllers/authController.js

```diff
@@ -64,3 +75,76 @@ exports.loginUser = (req, res) => {
         return res.status(200).json({message: "Successfully logged in", token})
   })
 }
+
+//This function is responsible to send email to a particular user with the token for password reset
+exports.forgotPassword = (req, res) => {
+  User.findOne({email: req.body.email} , (err, foundUser) => {
+    if (err || !foundUser) {
```

Comment:
This may not be an ideal way to handle this. Consider separating when there is an error and when the user is not found as the error may need to be reflected correctly